### PR TITLE
Add document describing how to add a custom Buildroot package

### PR DESCRIPTION
**Pull request by ConnorRigby**
Fixes https://github.com/nerves-project/nerves_system_rpi3/issues/89

### CODE REVIEW

**Comment by mobileoverlord** [comment]
Nice write up. I'm wondering if this should be on all the system readme files or in the Nerves docs with a link.

File: docs/Systems.md [hunk 1]
```diff
@@ -490,6 +490,52 @@ The [Buildroot user manual](http://nightly.buildroot.org/manual.html) can be
 very helpful, especially if you need to add a package. The various Nerves system
 repositories have examples of many common use cases, so check them out as well.
 
+## Adding a custom Buildroot Package
+Adding a package to your target Buildroot system is done by patching the 
+buildroot source tree. This process is exactly the same as adding a package
+to vanallia Buildroot. You can see their official guide
```
Comment (mobileoverlord) [inline docs/Systems.md]: vanallia  -> vanilla

**Comment by fhunleth** [comment]
@ConnorRigby Could you review this? I fixed markdown lint errors, fixed spelling, and made some other minor updates. If it's good with you, I'd like to merge.

**Comment by ConnorRigby** [comment]
@fhunleth it looks good to me! Thanks!

**Pull request merged**